src/init.c

```diff
@@ -614,13 +614,13 @@ _dispatch_bug(size_t line, long val)
 			_dispatch_build, (unsigned long)line, val);
 }
 
+#if HAVE_MACH
```

Comment:
the prototype in internal.h is already guarded this way, but agree that that seems wrong. This is currently only used in event_kevent.c (so could be guarded by DISPATCH_EVENT_BACKEND_KEVENT) but it is really intended to be a generic facility that can be used anywhere, so the better fix is probably to remove the ifdef of the prototype in internal.h